src/main/java/DukeBody/DataBase.java

```diff
@@ -0,0 +1,87 @@
+package dukebody;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Scanner;
+
+import duketask.Task;
+
+/**
+ * represents a database to query from and update to
```

Comment:
I think it would be good to capitalize the first letter of a sentence for JavaDoc comments.